packages/components/src/index.ts

```diff
@@ -201,6 +201,7 @@ export {
 } from './slot-fill';
 export { default as __experimentalStyleProvider } from './style-provider';
 export { ZStack as __experimentalZStack } from './z-stack';
+export { ProgressBar } from './progress-bar';
```

Comment:
As I also suggested earlier [here](https://github.com/WordPress/gutenberg/pull/61062#discussion_r1615858936), can we move this up to follow the pre-existing alphabetical order?

Reply:
Ah, sure 👍🏻 will keep that in mind as well. Thanks for the reminder.

Reply:
Done: https://github.com/WordPress/gutenberg/pull/61062/commits/43e1a57eb2bc2cea2f7bb6639f293de4e700c1f0.